### ISSUE TITLE
delete-installation infinite loop

### ISSUE THREAD

**Issue by chrisjs** [pivotal-cf/om#60]
it appears when calling delete-installation it loops through all the installation log endpoints. in the event it hits an install that was orphaned in the "RUNNING" state for whatever reason, when attempting to fetch the logs for that install, it seems like an infinite loop occurs, for example:

`X.X.X.X - opsman [07/Apr/2017:04:24:51 +0000] "POST /uaa/oauth/token HTTP/1.1" 200 2354 "-" "Go-http-client/1.1"
X.X.X.X - - [07/Apr/2017:04:24:52 +0000] "GET /api/v0/installations/155 HTTP/1.1" 200 31 "-" "Go-http-client/1.1"`

if you try to access the installation log of 155 via the ops manager UI, i.e.: https://pcf.host.com/installation_logs/155 you get redirected to /install then redirected back to the dashboard and the error message "Install is not in progress" appears and the loop to obtain those logs continues and the delete-installation process does not complete.

i ran into this on OpsManager v1.8.8.0 but not sure if it affects other versions. we are currently trying to move off the opsman GEM for rebuilding envs so this is a bit of a blocker for us

i am using the latest 0.21.0 version of "om"

**Comment by zachgersh** [comment]
@chrisjs so - interesting that you have an installation that got orphaned in a running state. I am actually unsure how we would work around that from an `om` perspective as that seems like an ops-manager bug (we just attempt to attach to any running installation). We might need to send this over to ops-manager.

**Comment by chrisjs** [comment]
yea not sure - we have had ops manager stability issues since 1.6 to the point where we were rebooting 10+ envs on a nightly crontab mainly due to memory issues. over the past few weeks we have dropped 1.6 & 1.7 leaving 1.8-1.10 deployments. maybe these running states got orphaned during the automated reboot process or maybe when we had to kill off tempest because the host ran out of memory, I'm not sure. 

is there any way to manually remove these "RUNNING" entries as a work around for the time being?
